apprentice/css/basics.md

```diff
@@ -0,0 +1,54 @@
+# CSS Basics
+
+## Intro
+
+When HTML was first invented, there were few ways to add style or change the way the elements looked. Inline tags, such as `color=` or `font=` were available, but missed the point of HTML being strictly for markup and not style.
+
+CSS (Cascading Style Sheets) was released in 1996 as a way to add styles to HTML elements. It uses an ingenious method of cascading styles, or having styles override other styles in a predictable way. This allows you to easily reuse styles for multiple elements.
+
+There is a lot to learn in CSS, and you will only gain the necessary skills through practice, trial and error, and making mistakes.
+
+This requirement will cover the three ways to add CSS to your HTML as well as a few selectors and attributes. It will also show you the potential for CSS.
+
+## Suggested Learning
+
+- Check out the following examples of awesome CSS:
+  - [CSS Zen Garden](http://csszengarden.com) - Check out several of the designs. They all use the same HTML file.
+  - [CSS-only Game](https://codepen.io/electerious/pen/ZeammP)
+  - [CSS-only Cube](https://codepen.io/hrtzt/pen/JdYaEZ)
+  - [CSS-only Page Turn](https://codepen.io/fbrz/pen/whxbF)
+  - [CSS-only Countdown Clock](https://codepen.io/kindofone/pen/DkhAz)
+
+- [How to add CSS to an HTML document](https://www.w3schools.com/css/css_howto.asp) - Also talks about cascading orders
+- [CSS Syntax](https://www.w3schools.com/css/css_syntax.asp) - Also talks about basic selectors
+- [CSS Backgrounds](https://www.w3schools.com/css/css_background.asp)
+- [CSS Text](https://www.w3schools.com/css/css_text.asp)
+- [CSS Width & Height](https://www.w3schools.com/css/css_dimension.asp)
+
+- Colors
+  - [Basic CSS Colors](https://www.w3schools.com/css/css_colors.asp) - Covers named, hex, and RGB colors
+  - [Advanced CSS Colors](https://www.w3schools.com/css/css3_colors.asp) - Covers RGBA, HSL(A), and Opacity
+- Units
+  - [CSS Measurement Units](https://www.w3schools.com/cssref/css_units.asp) - Focus on `px`, `%`, `em`, `rem`, `vw`, `vh`
+  - [More Measurement Units Info](https://www.tutorialspoint.com/css/css_measurement_units.htm)
+
+_Notice something interesting or strange about those examples? Ask your mentor about HTML and CSS pre-processors!_
+
+## Assignments
```

Comment:
The connotation I want to give is that the participant has to complete the assignments before earning the badge. Practice just indicates something you do to get better, not something you do to pass off. If you have better verbiage that fits that description I'd be interested to hear it.